windows-no-admin.md

```diff
@@ -72,4 +72,4 @@ To run it, save your modifications, go to the directory containing it, right cli
 python name-of-the-script.py
 ```
 
-In case you run into a problem, you can [open an issue](https://github.com/openfisca/openfisca-france/issues/new) or send an email to `contact@openfisca.fr`.
+In case you run into a problem, please [open an issue](https://github.com/openfisca/openfisca-core/issues/new).
```

Comment:
Keep email option and set it to `contact@openfisca.org`? 
Why would we close this door to new users for example?

Reply:
Because:

1. Do you know who reads that email address? 🙂
2. Issues allow for distribution of the support load.
3. How many email contacts have we received through that door in the last three years?

Reply:
I absolutely agree with _1._ in that I'm tired of receiving spam. 😄 

Regarding the change itself, seems to me it is out of scope, and you state it yourself @MattiSG both in the pull request description and in https://github.com/openfisca/openfisca-doc/pull/146/files#r191969020

Reply:
Yes. I will use less knights when the time to review is less than a week :wink:

Reply:
Ok, the email address wasn't really working here.